x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/aggregation/ValuesBytesRefAggregator.java

```diff
@@ -313,20 +335,21 @@ public void enableGroupIdTracking(SeenGroupIds seen) {
          * groups. This is the implementation of the final and intermediate results of the agg.
          */
         Block toBlock(BlockFactory blockFactory, IntVector selected) {
-            nextValues.prepareForEmitting(selected);
-            if (OrdinalBytesRefBlock.isDense(firstValues.size() + nextValues.hashes.size(), bytes.size())) {
-                return buildOrdinalOutputBlock(blockFactory, selected);
-            } else {
-                return buildOutputBlock(blockFactory, selected);
+            try (PreparedForEmitting prepared = nextValues.prepareForEmitting(selected)) {
+                // TODO restore this ordinal https://github.com/elastic/elasticsearch/issues/138705
+                // if (OrdinalBytesRefBlock.isDense(firstValues.size() + nextValues.hashes.size(), bytes.size())) {
+                // return buildOrdinalOutputBlock(blockFactory, selected, prepared);
+                // } else {
+                return buildOutputBlock(blockFactory, selected, prepared);
+                // }
```

Comment:
I mentioned this in the PR description - I think this may be a blocker to merging instead of a TODO for later. Not entirely sure.